File: Main-Server/Main-Server/Models/SubUser.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;
using System.Runtime.CompilerServices;

namespace Main_Server.Models
{
    public class SubUser
    {
        public string UserId { get; set; }
  
        public string SubId { get; set; }

        public string Name { get; set; }

        public string Surname { get; set; }

        public string Image { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public int PIN { get; set; }

        public List<Movie> Movies { get; set; }

        public List<Movie> FavoriteFilms { get; set; }

        public Movie LastWatched { get; set; }

        public List<Movie> FinishedMovies { get; set; }

        public List<Censor> Censors { get; set; }

    }
}

File: TVseries/lib/src/models/subUser.dart
import 'package:tv_series/src/models/censor.dart';
import 'package:tv_series/src/models/movie.dart';

class SubUser {
  int CustomerId=0;

  String Name;

  String Surname;

  String Image;

  String Title;

  String Description;

  int? PIN;

  int LastWatchedId=0;

  Movie? LastWatched;

  List<Movie>? Movies;
  
  List<Movie>? FavoriteMovies ;

  List<FinishedMovie>? FinishedMovies ;

  List<Censor>? Censors;

  SubUser(this.CustomerId, this.Name, this.Surname, this.Image, this.Title, this.Description, this.PIN);
}

class FinishedMovie {
}
